jdaviz/configs/mosviz/plugins/parsers.py

```diff
@@ -236,14 +236,14 @@ def mos_spec1d_parser(app, data_obj, data_labels=None):
 
 @data_parser_registry("mosviz-spec2d-parser")
 def mos_spec2d_parser(app, data_obj, data_labels=None, add_to_table=True,
-                      show_in_viewer=False):
+                      show_in_viewer=False, extension=1, transpose=False):
```

Comment:
`ext` is more in line with `astropy.io.fits` API.
```suggestion
                      show_in_viewer=False, ext=1, transpose=False):
```